src/client/errors.ts

```diff
@@ -88,17 +63,17 @@ export enum ChannelError {
     /**
      * Indicates that a channel of a provided ID does not exist.
      */
-    ChannelWithIdDoesNotExist = 'ChannelWithIdDoesNotExist'
+    ChannelWithIdDoesNotExist = 'ChannelError:ChannelWithIdDoesNotExist'
 }
 
 /**
  * Error codes relating to OpenFin windows.
  */
 export enum IdentityError {
     /**
-     * Indicates that a window with a provided OpenFin Identity cannot be found.
+     * Indicates that no window with a provided OpenFin Identity is registered with the FDC3 service.
      */
-    WindowWithIdentityNotFound = 'WindowWithIdentityNotFound'
+    WindowWithIdentityNotFound = 'IdentityError:WindowWithIdentityNotFound'
```

Comment:
Make this less window-specific, eg `ConnectionNotFound`

Reply:
✅

Reply:
Based on outcomes of SERVICE-737, kinda changed my mind about this one. For the client API, I think it makes more sense if we just leave everything in terms of 'windows'.

Sorry :-)

Reply:
On intuition I've left the enum as `ConnectionError`, but then the specific error code is about a windows